certified-connectors/jbhunt/apiProperties.json

```diff
@@ -196,13 +150,6 @@
             "Find_orders"
           ]
         }
-      },
-      {
-        "templateId": "dynamichosturl",
```

Comment:
@vechall can you please explain why did you remove the dynamichosturl?

Reply:
@v-EgorKozhadei, Initially we provided the option to use different URLs based on user selection.